jsontableschema_sql/storage.py

```diff
@@ -14,7 +17,7 @@
 
 # Module API
 
-class Storage(object):
+class Storage(Storage):
```

Comment:
perhaps we import Storage as BaseStorage or something so we don't reuse the exact name, causing confusion and ambiguity?